test/e2e/ec2/instance.go

```diff
@@ -145,6 +145,20 @@ func (e *InstanceConfig) Create(ctx context.Context, ec2Client *ec2.Client, ssmC
 	}, nil
 }
 
+// DisableSourceDestCheck disables the source/destination check for the given instance.
+func DisableSourceDestCheck(ctx context.Context, ec2Client *ec2.Client, instanceID string) error {
```

Comment:
so by default an ec2 instance blocks the packet if the src/dest is outside the vpc(?) ?

Reply:
yeah

> You can enable or disable source/destination checks, which ensure that the instance is either the source or the destination of any traffic that it receives. Source/destination checks are enabled by default. You must disable source/destination checks if the instance runs services such as network address translation, routing, or firewalls.